share/install_gems/Gemfile

```diff
@@ -46,6 +46,7 @@ if RUBY_VERSION < '2.3'
 else
     gem 'zendesk_api'                     # sunstone
     gem 'rqrcode'                         # sunstone
+    gem 'webauthn'                        # sunstone
```

Comment:
the latest version requires RUBY_VERSION >= 2.4.0
https://rubygems.org/gems/webauthn/versions/2.2.0